app/src/main/java/org/wordpress/pioupiou/login/LoginActivity.java

```diff
@@ -221,9 +274,13 @@ public void onUrlChecked(OnURLChecked event) {
     @Subscribe(threadMode = ThreadMode.MAIN)
     public void onAuthenticationChanged(OnAuthenticationChanged event) {
         if (event.isError()) {
-            // TODO
-            Timber.i("onAuthenticationChanged error");
-            Toast.makeText(this, event.error.message, Toast.LENGTH_SHORT).show();
+            if (event.error.type == NEEDS_2FA) {
```

Comment:
This would be better without the static import I think, so it would be `AuthenticationErrorType.NEEDS_2FA`.